research/DandA/data.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: detect
```

Comment:
```suggestion
layout: default
```

Reply:
@mauzey1 - part of the goal of this commit was to add the links (as in the Cloud Feedbacks Project). Can I keep this set as `detect.md`?

Same for `index.md`, `people.md`, and `publications.md`

Reply:
There is currently no layout called detect in the _layout directory.  Will that be part of this commit?

Reply:
Sorry @mauzey1 - I thought it was in the commit. It is added now.